external/dockerfile_functions.sh

```diff
@@ -498,7 +498,7 @@ print_testInfo_env() {
     local test=$1
     local test_tag=$2
     local OS=$3
-    local version=$4
+    local version=$3
```

Comment:
Both `OS` and `version` are assigned `$3`, we need to fix it here. Thanks.

Reply:
Yes, I do agree, that's what I suggested that we might need another issue for this OS_TAG parameter.